### PR TITLE
Migrate from CircleCI 1.0 to 2.0

### LINKED ISSUE #135: Move CircleCI from version 1.0 to version 2.0
After August 31, 2018, CircleCI 1.0 will no longer be available for Linux and macOS users. You can find guides for transitioning from 1.0 to 2.0 and a full timeline on planned changes

Official information
https://circleci.com/blog/sunsetting-1-0/

### PR DESCRIPTION
**Pull request by hectorerb**
Signed-off-by: Hector Rondon <hrondon@teclib.com>

### Changes description
- Migrate build, test and deploy to circleCI 2.0
- Update Header license (Copyright, email author)

### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### Estimated time

|Assignee|:tomato:|
|:---|:---:|
|@hectorerb |14|

### References

Closes #135 
Related https://github.com/flyve-mdm/ios-inventory-agent/pull/129
Depends on #N/A

### CODE REVIEW

File: ci/scripts/set_locate.sh [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+
+#   Copyright © 2017 Teclib. All rights reserved.
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh]: 2017-2018

File: ci/scripts/set_locate.sh [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 3
+# of the License, or (at your option) any later version.
+#
+# flyve-mdm-ios is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# ------------------------------------------------------------------------------
+# @author    Hector Rondon
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh]: Email

File: ci/scripts/set_locate.sh [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+# @date      22/06/18
+# @copyright Copyright © 2017 Teclib. All rights reserved.
+# @license   GPLv3 https://www.gnu.org/licenses/gpl-3.0.html
+# @link      https://github.com/flyve-mdm/flyve-mdm-ios-agent
```
Comment (ajsb85) [inline ci/scripts/set_locate.sh]: New repo URL

**Review by ajsb85 — CHANGES_REQUESTED** [review]

File: ci/scripts/set_locate.sh (hunk 1 above)
Comment (hectorerb) [reply ci/scripts/set_locate.sh]: done

File: ci/scripts/set_locate.sh (hunk 2 above)
Comment (hectorerb) [reply ci/scripts/set_locate.sh]: Done

File: ci/scripts/set_locate.sh (hunk 3 above)
Comment (hectorerb) [reply ci/scripts/set_locate.sh]: Done

**Review by ajsb85 — APPROVED** [review]

**Pull request merged**